degica/rails_drivers
# Fix CI and update node dependencies

The CI started magically failing, in part because we hadn't pinned the version of Rubocop we were using. I've updated Rubocop and pinned the version so that we can expect consistent CI results hereon.

### How to test

Just check that the CI is actually running rubocop and tests :wink:. Can try to run locally too if you want.
